### DISCUSSION TITLE
configure location of __mocks__ folder ?

### DISCUSSION THREAD

**Discussion by isimmons** [vitest-dev/vitest#5214] [Q&A] [answered]
I see in jest there is a "roots" config option where I could list "myproject/tests/__mocks__" but I haven't found any such option in vitest. Is there a way to move __mocks__ into my tests directory?

**Comment by sheremet-va** [comment, marked as answer]
No, it will use your root option instead

**Reply by isimmons** [reply]
Thanks,
Do you mean this should work so I can put `__mocks__` inside the tests directory?

```js
import { defineConfig } from "vitest/config";

export default defineConfig({
  test: {
    root: "./tests",
  },
});
```

**Reply by isimmons** [reply]
Ah yes it does. I had forgot to put vi.mock('fs') in my test file and thought it wasn't using the mock from `__mocks__` but after fixing my mistake this is working great.
Thanks
